src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figures {

```

Comment:
According to the checklist, avoid using verbs for class/interface names. The name of the class 'Figures' should be 'Figure'.